### DISCUSSION TITLE
Individual file type icons

### DISCUSSION THREAD

**Discussion by Freeman-ab** [sumatrapdfreader/sumatrapdf#4496] [Ideas]
Is it possible to add individual file type (PDF, epub, mobi, cbz, cbr, DjVu, XPS, CHM) icons like it is on 7-zip? I made a mockup image down below.

![demo](https://github.com/user-attachments/assets/264765fd-058c-469f-97e2-978b2842a99e)

**Comment by GitHubRulesOK** [comment]
there are (have been many suggestions) but only one or 2 are relevant to perceived installable preview able types
last updated in https://github.com/sumatrapdfreader/sumatrapdf/pull/1592
![image](https://github.com/user-attachments/assets/42e15b9e-7fc1-47d3-89fd-d8c0e5368706) ![image](https://github.com/user-attachments/assets/9662e1e6-0644-4120-b33f-021c9f65a027)

Normally **unless the preview is not possible** (password protected etc.) they will not always be seen 
![image](https://github.com/user-attachments/assets/323f2904-69d1-4122-8a8c-70a8c377d2fe)


NOTE CBX is a nominal term for Comic bookX so There can be OTHER real CBX  files where SumatraPDF may or may not show those icons in different cases. the icon should be confined to cbr, cbt, cbz or any "known" compressed archive types that SumatraPDF can read

![image](https://github.com/user-attachments/assets/039a5fb8-97d0-461d-ab42-5f39e57c7e09)

**Comment by Freeman-ab** [comment]
Yes i figured it out. I didn't make a search tbh, sorry for that
